src/PKSim.Core/CoreConstants.cs

```diff
@@ -1335,9 +1335,9 @@ public static class ObservedData
 
          public static IReadOnlyList<string> DefaultProperties = new List<string>
          {
-            OSPSuite.Assets.ObservedData.ORGAN,
-            OSPSuite.Assets.ObservedData.COMPARTMENT,
-            OSPSuite.Assets.ObservedData.MOLECULE,
+            OSPSuite.Assets.ObservedData.Organ,
```

Comment:
@abdelr @georgeDaskalakis 
Why was this renamed? Constants are normally ALL_CAPS. Let's make sure we revert this in the next update in core